src/ducks/account.ts

```diff
@@ -67,4 +77,28 @@ const accountSlice = createSlice({
   },
 });
 
+// TODO - make thunk methods consistent (eg. use createAsyncChunk or not)
+// leaving like this to compare
+export const fetchAccountTxHistory = (accountId: string) => {
+  return async (dispatch: AppDispatch) => {
+    const server = new StellarSdk.Server("https://horizon-testnet.stellar.org");
+    return (
+      server
```

Comment:
`wallet-sdk` `DataProvider` has a method `fetchPayments()` which we could use here: https://github.com/stellar/js-stellar-wallets/blob/master/src/data/DataProvider.ts#L182

Reply:
@quietbits  Are you thinking we should default to using wallet-sdk when we can? And only if something isn't there then use js-sdk?

Reply:
Yes, I think we should use `wallet-sdk` where we can. That way we can be more consistent.